src/cargo/util_schemas/mod.rs

```diff
@@ -5,4 +5,5 @@
 //! Any logic for getting final semantics from these will likely need other tools to process, like
 //! `cargo metadata`.
 
+pub mod core;
```

Comment:
Although it is just moving things around, `core` is not too meaningful as a name.

Reply:
Got a better idea?
- I considered `common` and `misc` but figured those were the same or worse
- I considered the root but then that gets messy
- I considered `manifest` but these are general beyond that

Reply:
Or remove the entire `core` and expose every type to root level?

Reply:
Thats the "messy" part.  I feel like it'd more help users reading the docs to not have a large dumping ground in the root of types that are secondary to what they are doing.